### PR TITLE
BDOG-693 Update dependencies & add type to repository.yaml

### PR DESCRIPTION
**Pull request by chotai**

### CODE REVIEW

File: scripts/bin/create.py [hunk 1]
```diff
@@ -191,6 +190,14 @@ def move_files_to_dist(dirs, src, dst):
...
+    filename = "%s/repository.yaml" % repo
+    if os.path.exists(filename):
+        with open(filename, "a") as f:
+            f.write("\ntype: %s" % repository_type)
+        f.close()
+
+
 def clone_repo(repo, github_token):
     command = 'git clone https://%s@github.com/hmrc/%s' % (github_token, repo)
     print(f"cloning: {command}")
```
Comment (christopherjturner) [inline scripts/bin/create.py:203]: nothing to do with your change but does this line basically print out the github token given its part of the url?

**Review by christopherjturner — DISMISSED** [review]

File: scripts/bin/create.py [hunk 2]
```diff
@@ -222,11 +229,16 @@ def push_repo(project_name, default_branch):
...
+    parser.add_argument('--github', action='store_true', help='deprecated')
+    parser.add_argument('--with-mongo', action='store_true',
+                        help='Does your service require Mongo? This only available if the repository is of type "BACKEND"')
+    parser.add_argument('--github-token', help='The GitHub token authorised to push to the repository')
+    parser.add_argument('--default-branch', default='master',
+                        help='The default branch name for the GitHub repository when pushing changes')
     args = parser.parse_args()
 
-    create_service(args.REPOSITORY, args.type, args.github, args.with_mongo, args.github_token, args.default_branch)
+    create_project(args.REPOSITORY, args.type, args.dry_run, args.with_mongo, args.github_token, args.default_branch)
```
Comment (colin-lamed) [inline scripts/bin/create.py:244]: does the deprecated `args.github` need to be a fallback when `args.dry_run` is not set?

Comment (chotai) [reply scripts/bin/create.py:244]: Previously `--github` had to be present to sync with github but now this behaviour is on my default. B&D clone directly from github so can't remove `--github` just yet.

File: scripts/bin/create.py (hunk 1 above)
Comment (chotai) [reply scripts/bin/create.py:203]: good spot - yes it does. Will remove

**Review by colin-lamed — APPROVED** [review]

**Pull request merged**